_includes/post-download-script.html

```diff
@@ -28,6 +28,26 @@
                 file_ext = 'zip';
                 package_name = 'Zowe Python Client SDK'
                 break;
+            case 'zen-windows':
+                filename_prefix = 'zowe-install-wizard-windows';
+                file_ext = 'exe';
+                package_name = 'zowe-install-wizard'
+                break;
+            case 'zen-mac':
+                filename_prefix = 'zowe-install-wizard-mac';
+                file_ext = '-x64.dmg';
+                package_name = 'zowe-install-wizard'
+                break;
+            case 'zen-rpm':
+                filename_prefix = 'zowe-install-wizard-rpm';
+                file_ext = '-1.x86_64.rpm';
```

Comment:
I'm not sure why the -1 is in the filename, but just filter on x86_64.rpm please in case it changes.